### ISSUE TITLE
Allow case-insensitive lookup

### ISSUE THREAD

**Issue by acarl005** [pyrossh/rust-embed#249]
For many use-cases, case-insensitivity is very useful. It would be convenient to have an attribute to allow this.

**Comment by pyrossh** [comment]
Yes might be useful. Would you be interested in a PR for this feature?
